tests/by-util/test_ls.rs

```diff
@@ -3678,3 +3730,55 @@ fn test_ls_dired_complex() {
     println!("Extracted filenames: {:?}", filenames);
     assert_eq!(filenames, vec!["a1", "a22", "a333", "a4444", "d"]);
 }
+
+#[test]
```

Comment:
When running the steps from this test manually I noticed two differences in the output to the output from GNU ls: the "DIRED" line is missing and the two last values in the "SUBDIRED" line are different:
```
uutils ls
//SUBDIRED// 2 7 291 298 291 297

GNU ls
//SUBDIRED// 2 7 291 298 313 319
```

Reply:
yeah, it is normal given our output